src/naquibalam/A13_StringFunction.java

```diff
@@ -0,0 +1,62 @@
+package naquibalam;
+
+import java.util.Scanner;
+
+public class A13_StringFunction {
+	
+	void getAllDuplicateChar() {
+		Scanner scanner1 = new Scanner(System.in);
+		System.out.println("Please enter the String for which you want duplicate characters : ");
+		String stringToUse = scanner1.next();
+		System.out.println("Print only first duplicate character (true/false) : ");
+		boolean printOnlyFirst = scanner1.nextBoolean();
+		if (printOnlyFirst) {
+			System.out.print("First duplicate character in string '" + stringToUse + "' is : ");
+		} else {
+			System.out.print("All duplicate character(s) in string '" + stringToUse + "' : ");
+		}
+		for (int i=0; i <= stringToUse.length()-1; i++) {
+			int firstIndexOf = stringToUse.indexOf(stringToUse.charAt(i));
+			int lastIndexOf = stringToUse.lastIndexOf(stringToUse.charAt(i));
+			if ((firstIndexOf != lastIndexOf) && (lastIndexOf == i)) {
+				System.out.print(stringToUse.charAt(i) + " ");
+				if (printOnlyFirst) {
+					break;
+				}
+			}
+		}
+		scanner1.close();
+		System.out.println("");
+	}
+	
+	void getAllUniqueChar() {
+		Scanner scanner2 = new Scanner(System.in);
```

Comment:
taking input from service method create code logic bit heavy so try to take input from only main method and pass those input in methods.